string action
detect_mechanical_parts/DetectionMsg[] boundingBoxes
int64 timestamp